source/playbooks/notifications-and-updates.rst

```diff
@@ -1,11 +1,20 @@
 Notifications and Updates
 =========================
 
-To help you keep track of your runs, tasks, and statuses, a daily digest is sent via Direct Message in Mattermost.
+There are multiple ways to receive updates and notifications.
+
+Follow runs and playbooks
+-------------------------
+
+You can choose to follow specific playbooks so that you'll receive updates on important events such as when a run is started and finished, as well as status updates and retrospetive updates every time that playbook is used for a run. This is a good option if you're a interested in all instances of a specific workflow, such as an outage resolution playbook.
+
+As a stakeholder, you can also choose to follow only a specific run. This means that while you don't have to participate in the run, you'll receive updates and can decide what, if any, actions to take. This is a good option if you're only interested in a single instance of a procedure, for example, you may want to only follow customer onboarding for one specific customer instead of all the onboarding runs.
```

Comment:
```suggestion
As a stakeholder, you can also choose to follow only a specific run. This means that while you don't have to participate in the run, you'll receive updates and can decide what, if any, actions to take. This is a good option if you're only interested in a single instance of a procedure. For example, you may want to only follow customer onboarding for one specific customer instead of all onboarding runs.
```